docs/user-guide/cli-swreqplugins.md

```diff
@@ -11,7 +11,6 @@ Before you use Zowe&trade; CLI plug-ins, complete the following steps:
 | [IBM z/OS FTP Plug-in for Zowe CLI](cli-ftpplugin.md) | <ul> <li>Ensure that z/OS FTP service is enabled and configured with `JESINTERFACELEVEL` = 2.</li> <li>FTP over SSL is recommended.</li> </ul> |
 | [IBM IMS Plug-in for Zowe CLI](cli-imsplugin.md) | <ul><li>Ensure that [IBM® IMS™ v14.1.0](https://www.ibm.com/support/knowledgecenter/en/SSEPH2_14.1.0/com.ibm.ims14.doc/ims_product_landing_v14.html) or later is installed and running in your mainframe environment.</li><li> Configure [IBM® IMS™ Connect](https://www.ibm.com/support/knowledgecenter/en/SSEPH2_13.1.0/com.ibm.ims13.doc.ccg/ims_ct_intro.html).</li> <li>Configure [IBM IMS Operations APIs](https://github.com/zowe/ims-operations-api) to enable communication between the CLI and the IMS instance. </li></ul> |
 | [IBM MQ Plug-in for Zowe CLI](cli-mqplugin.md) | <ul><li>Ensure that [IBM® MQ™ v9.1.0](https://www.ibm.com/support/knowledgecenter/en/SSFKSJ_9.1.0/com.ibm.mq.pro.doc/q121910_.htm) or later is installed and running in your mainframe environment. Please read this blog for more information: [Exposing the MQ REST API via the Zowe API Mediation Layer](https://developer.ibm.com/messaging/2019/05/17/exposing-the-mq-rest-api-via-the-zowe-api-mediation-layer/) </li></ul>|
-| [Secure Credential Store Plug-in for Zowe CLI](cli-scsplugin.md) | <ul> <li> (Linux only) Install libsecret on your computer. </li></ul> |
-
+| [Secure Credential Store Plug-in for Zowe CLI](cli-scsplugin.md) | <ul> <li> **(Graphical Linux)** Install `gnome-keyring` and `libsecret` on your computer. </li> <li>There are additional requirements for headless Linux systems. See the [SCS plug-in Readme](https://docs.zowe.org/stable/user-guide/cli-swreqplugins.html) for details.</li> </ul> |
```

Comment:
Should the link URL here be https://github.com/zowe/zowe-cli-scs-plugin/blob/master/README.md ?